Training/ImageClassifier_Tumour.py

```diff
@@ -11,27 +12,18 @@
 from torch.utils.data import DataLoader
 from QA.Normalization.Colour import ColourNorm
 from Model.ConvNet import ConvNet
-import sys
 
 
 config = toml.load(sys.argv[1])
-#config = toml.load('./config_files/preprocessing/trainer_tumour_convnet.ini')  example of local file.
-
-# stuff to try:
-# 1) create csv files from the 48 training examples, and train on it. Get a model.
-# 2) use said model to test on the remaining 10 examples, and make sure that those examples you get their labels.
-# 3) see what happens when predicting a new slide?? with no label at all. Can I use contourstocsv?
+# config = toml.load('/home/mikael/Dropbox/M/PostDoc/UCL/Code/Python/DigitalPathologyAI/Training/config_files/preprocessing/trainer_tumour_convnet.ini')  # example of config file
 
 ########################################################################################################################
-#                                      1. Pre-processing: create csv files
-force_preprocessing = 0
-# TODO: make an option to only make non-existing ones!
+#                                      1. Pre-processing: create npy files
 
-if force_preprocessing:
-    preprocessor = PreProcessor(config)
-    preprocessor.ContoursToCSV()
-
-# preprocessor.LabelToCSV()
+preprocessor = PreProcessor(config)
+preprocessor.AnnotationsToNPY(overwrite=False)
+id_dict = dict(zip(preprocessor.config['CRITERIA']['id_internal'], preprocessor.config['INTERNAL']['WSI_processing_index']))
```

Comment:
I think we should use the dataframe ID layer here, as it's more robust (only one place it can crash is in the loading from server)